instrumentation/jedis/jedis-1.4/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jedis/v1_4/JedisConnectionInstrumentation.java

```diff
@@ -118,4 +132,85 @@ public static void stopSpan(
       JedisRequestContext.endIfNotAttached(instrumenter(), context, request, throwable);
     }
   }
+
+  @SuppressWarnings("unused")
+  public static class SendCommandWithProtocolNoArgsAdvice {
+
+    @Advice.OnMethodEnter(suppress = Throwable.class)
+    public static void onEnter(
+        @Advice.This Connection connection,
+        @Advice.Argument(0) Object command,
+        @Advice.Local("otelJedisRequest") JedisRequest request,
+        @Advice.Local("otelContext") Context context,
+        @Advice.Local("otelScope") Scope scope) {
+      Context parentContext = currentContext();
+      Protocol.Command cmd = null;
+      // It must be a Protocol.Command in redis 2.7.2
+      if (command instanceof Protocol.Command) {
```

Comment:
Add line 149~151 to `SendCommandNoArgsAdvice.onEnter` and adjust the param of `@Advice.Argument(0) Protocol.Command` command to `@Advice.Argument(0) Object command.`. Is it better?

Reply:
No, I've tried it. advice doesn't work if we set args(0) to `Protocol.Command`. The type transform will not be executed automatically.

Reply:
> No, I've tried it. advice doesn't work if we set args(0) to `Protocol.Command`. The type transform will not be executed automatically.

Use an object type to receive the param and them write some codes to transform them manually.